cmake/package-functions.cmake

```diff
@@ -11,7 +11,7 @@ file( WRITE ${scripts_write_dir}/postinst
 set -e
 
 do_ldconfig() {
-  echo ${CPACK_PACKAGING_INSTALL_PREFIX}/${LIB_INSTALL_DIR} > ${ld_conf_file} && ldconfig
+  echo /opt/rocm/${LIB_INSTALL_DIR} > ${ld_conf_file} && ldconfig
```

Comment:
Could you explain the change a little bit? Why it is going to /opt/rocm/?

Reply:
This change is not required. Since itseems the function write_rocm_package_script_files() is not used/called. However the corresponding change is in library/CMakelists.txt - ROCFFT_CONFIG_DIR is passed to rocm_package_create(). So reverting this change.